Change contents label according to 505 indicator

marlo-longley opened the pull request:
Fixes https://github.com/sul-dlss/SearchWorks/issues/3103

See https://www.loc.gov/marc/bibliographic/bd505.html for indicator/status names

<img width="1134" alt="Screenshot 2023-04-14 at 10 51 22" src="https://user-images.githubusercontent.com/1328900/232078432-dc833fc0-6e96-499b-a16a-9bdc2f531cfc.png">

cbeer commented:
I wonder if we also want to map indicator1=1 to "Partial contents".

I mean, I don't know what "Partial contents" means either, but the description of "Incomplete contents" seems pretty library-workflow-centric, doesn't it?

@saseestone ?

marlo-longley commented:
I rebased to master and the new rubocop rules, and changed "Incomplete" to "Partial" contents in the label.

Jenkins failed with  
```
ERROR: Error fetching remote repo 'origin'
hudson.plugins.git.GitException: Failed to fetch from https://github.com/SUL-DLSS/searchworks_traject_indexer.git
```
